src/components/core/Button/Button.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import classes from './Button.module.scss';
 
 interface ButtonProps {
-  variant?: 'secondary';
+  variant?: string | 'secondary';
```

Comment:
Set this as just string or give it specific value, it shouldn't be `string | 'secondary'`